src/js/modules/RoutingApp.js

```diff
@@ -51,11 +51,13 @@ const RoutingApp = () => {
   return (
     <Provider store={store}>
       <ConnectedRouter history={appHistory}>
+         <PageLayout>

```

Comment:
I've tried this before, and it messes up the routing. Try it; if it's messed up, then delete it. We'll have to figure this out at a later date. (It should just be some arrangement of `ConnectedRouter`, `Switch`, and `PageLayout`.